proiect/src/order/OrderItem.java

```diff
@@ -0,0 +1,72 @@
+package order;
+
+
+import java.util.Scanner;
+
+import business.Product;
+
+public class OrderItem {
+
+	private Product product;
+	
+	private int quantity;
+	
+	/**
+	 * @param product
+	 * @param quantity
+	 */
+	public OrderItem(Product product, int quantity) {
+		super();
+		this.product = product;
+		this.quantity = quantity;
+	}
+	
+	/**
+	 * @param product
+	 * @param quantity
+	 */
```

Comment:
javadoc params not suitable anymore, you can update them

Reply:
Resolved in issue #7